docker/clone-and-preflight.ts

```diff
@@ -64,15 +64,24 @@ if (projectUsesPostgresql) {
   //
   // Example script:
   // https://github.com/upleveled/preflight-test-project-next-js-passing/blob/e65717f6951b5336bb0bd83c15bbc99caa67ebe9/scripts/alpine-postgresql-setup-and-start.sh
-  const postgresUid = Number((await execa`id -u postgres`).stdout);
-  await execa({
+  const postgresProcess = await execa({
     // postgres user, for initdb and pg_ctl
-    uid: postgresUid,
+    uid: Number((await execa`id -u postgres`).stdout),
     // Show output to simplify debugging
-    stdout: 'inherit',
-    stderr: 'inherit',
+    stdout: ['inherit', 'pipe'],
+    stderr: ['inherit', 'pipe'],
```

Comment:
Changed to `['inherit', 'pipe']` for programmatic + terminal output. Print directly in the terminal with `inherit` and capture the process output with `pipe`.